browser/pages/confirm/confirm.html

```diff
@@ -453,6 +453,22 @@
       </div>
     </div>
 
+    <!-- Kompose -->
+    <div id="kompose-panel" class="panel panel-default panel2pxborder">
+      <div id="kompose-panel-heading" class="panel-heading panel-normal">
+           <div class="checkbox-container verticalLine">
+             <input id="kompose-checkbox" type="checkbox" ng-model="checkboxModel.kompose.selectedOption" aria-label="Toggle ngHide" class="vallign-middle" ng-true-value="'install'" ng-false-value="'detected'">
+           </div>
+        <div class="checkbox-container" ng-show="false">
+          <div id="arrow-kompose" class="arrow" ng-class="{'arrow-down':!checkboxModel.kompose.isCollapsed}" aria-label="Toggle ngHide"></div>
+        </div>
+        <div class="product-container">
+          <span id="kompose-name" class="product-name">{{checkboxModel.kompose.productName}}</span>
+          <span id="kompose-version" class="product-version">{{checkboxModel.kompose.version}}</span>
+          <div id="kompose-description">{{checkboxModel.kompose.productDesc}}</div>
+        </div>
+      </div>
+    </div>
```

Comment:
@pradeepto After this div we can add info message with link to https://kompose.io

``` html
<div class="help-block">
   <span class="pficon pficon-info"></span>
   <span>For additional information and documentation visit
      <a type="button" class="pointer" ng-click="confCtrl.download('https://kompose.io')">https://kompose.io</a>
   </span>
</div>
```

so final UI would look like

![kompose-ui](https://cloud.githubusercontent.com/assets/620330/24311094/dc6ed5ec-108f-11e7-94a2-f9d46218bee6.png)

@cdrage @pradeepto WDYT?